.goreleaser.yaml

```diff
@@ -33,9 +33,9 @@ builds:
     ldflags:
       - '-s'
       - '-w'
-      - '-X={{ .ModulePath }}/internal/version.Name=abc'
-      - '-X={{ .ModulePath }}/internal/version.Version={{ .Version }}'
-      - '-X={{ .ModulePath }}/internal/version.Commit={{ .Commit }}'
+      - '-X={{ .ModulePath }}/internal/version.name=abc'
```

Comment:
Curious to learn why we don't use the default flags in ` -s -w -X main.version={{.Version}} -X main.commit={{.Commit}} -X main.date={{.Date}} -X main.builtBy=goreleaser` in go releaser, see https://goreleaser.com/customization/builds/, I thought with the default flag, it will just override the buildinfo.Version() in `internal/version/version.go`.

Reply:
Never mind. I made an experiment. It won't override the buildinfo.Version.